airbyte-ci/connectors/metadata_service/lib/metadata_service/validators/metadata_validator.py

```diff
@@ -206,6 +206,34 @@ def validate_docker_image_tag_is_not_decremented(
     return True, None
 
 
+def validate_release_candidate_has_rc_suffix_in_version(
+    metadata_definition: ConnectorMetadataDefinitionV0, _validator_opts: ValidatorOptions
+) -> ValidationResult:
+    # Bypass validation for pre-releases
+    if _validator_opts and _validator_opts.prerelease_tag:
+        return True, None
+
+    is_release_candidate = get(metadata_definition, "data.releases.isReleaseCandidate")
+    if not is_release_candidate:
+        return True, None
+
+    docker_image_tag = get(metadata_definition, "data.dockerImageTag")
+    if docker_image_tag is None:
+        return False, "The dockerImageTag field is not set."
+    try:
+        parsed_version = semver.VersionInfo.parse(docker_image_tag)
+        has_rc_suffix = parsed_version.prerelease and "rc" in parsed_version.prerelease
+        if not has_rc_suffix:
+            return (
+                False,
+                "The dockerImageTag field should end with -rc as the connector is marked as a release candidate (releases.isReleaseCandidate).",
```

Comment:
```suggestion
                "The dockerImageTag field should have an -rc.<RC #> suffix as the connector is marked as a release candidate (releases.isReleaseCandidate). Example: 2.1.0-rc.1",
```

Just wanted it to be clear that there should be an rc number at the end, too.